packages/react-components-lab/src/hooks/useLocalStorage/types.ts

```diff
@@ -0,0 +1,9 @@
+interface InitialVal {
+  initialValue: unknown;
+}
+
+interface HookTypes extends InitialVal {
+  key: string;
+}
+
+export { InitialVal, HookTypes };
```

Comment:
```suggestion
export interface InitialVal {
  initialValue: unknown;
}

export interface HookTypes extends InitialVal {
  key: string;
}
```

You could do this instead, just a suggestion. I think that's how we do the export in other ones too.

Reply:
Thank you! Done